examples/nine-pt-stencil-solver/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(nine - pt - stencil - solver nine - pt - stencil - solver.cpp)
```

Comment:
Some weird formatting here. Could you remove the spaces and align the lines below ? It could be the same way it is in the `three-pt-stencil` example.

Reply:
fixed this. I used clang-format incorrectly. Using `make format` now.